src/OSPSuite.Presentation/IPresentationUserSettings.cs

```diff
@@ -55,6 +55,12 @@ public interface IPresentationUserSettings
       /// </summary>
       Color ChartDiagramBackColor { get; set; }
 
+      /// <summary>
+      ///    When dragging and dropping observed data folders to a chart
+      ///    assign the same color to curves coming from the same folder
+      /// </summary>
+      bool ColorGroupObservedDataFromSameFolder { get; set; }
```

Comment:
Type should probably be bool? 
So that you know if it was asked already or not. E.g. you need 3 states

Reply:
After discussing with Pavel we thought that if the color grouping is initially disabled and you get the option to toggle it on and off both through the observed data context manu and the settings, we could skip the dialog. I have aslso updated this in the description of the issue, where thie drag and dropping part of the feature is currently fully documented.

Reply:
I somewhat disagree with this. Why not tell the user that this is a feature available? This is also what we had discussed